pom.xml

```diff
@@ -142,6 +142,7 @@
   <modules>
     <module>cdap-common-unit-test</module>
     <module>cdap-api</module>
+    <module>cdap-app-templates</module>
```

Comment:
Don't need to add cdap-app-templates. It is built with when templates profile is enabled.